x-pack/plugins/ml/public/application/help_popover.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode, useEffect } from 'react';
+import {
+  EuiIcon,
+  EuiLink,
+  EuiLinkButtonProps,
+  EuiPopover,
+  EuiPopoverProps,
+  EuiPopoverTitle,
+  EuiText,
+} from '@elastic/eui';
+import './help_popover.scss';
+
+export const HelpPopoverButton = ({
+  children,
+  onClick,
+}: {
+  children: string;
+  onClick: EuiLinkButtonProps['onClick'];
+}) => {
+  return (
+    <EuiText size="xs">
+      <EuiLink onClick={onClick}>
+        <EuiIcon className="mlHelpPopover__buttonIcon" size="s" type="help" />
+
+        {children}
+      </EuiLink>
+    </EuiText>
+  );
+};
+
+export const HelpPopover = ({
```

Comment:
You should move this into a `help_popover` folder under `ml/public/application/components`

Reply:
Done in 9bd45470c45aad9d297240e97c28113202143df4